FMDB.podspec

```diff
@@ -7,7 +7,12 @@ Pod::Spec.new do |s|
   s.author = { 'August Mueller' => 'gus@flyingmeat.com' }
   s.source = { :git => 'https://github.com/ccgus/fmdb.git', :tag => "#{s.version}" }
   s.requires_arc = true
-  s.default_subspec = 'standard'  
+  s.default_subspec = 'standard'
+
+  s.subspec 'common' do |ss|
+    ss.source_files = 'src/fmdb/FM*.{h,m}'
+    ss.exclude_files = 'src/fmdb.m'
+  end
 
   # use the built-in library version of sqlite3
   s.subspec 'standard' do |ss|
```

Comment:
`s.subspec 'standard' do |ss|` should just have the `ss.dependency 'FMDB/common'` dependency instead of duplicating the source_files and exclude_files.